activation/post_test.go

```diff
@@ -256,10 +256,76 @@ func TestPostSetupManager_Stop_WhileInProgress(t *testing.T) {
 	req.LessOrEqual(status.NumLabelsWritten, uint64(opts.NumUnits)*cfg.LabelsPerUnit)
 
 	// Continue to create data.
-	req.NoError(mgr.StartSession(context.Background(), opts, goldenATXID))
+	req.NoError(mgr.StartSession(context.Background(), opts))
 
 	// Verify status.
 	status = mgr.Status()
 	req.Equal(PostSetupStateComplete, status.State)
 	req.Equal(uint64(opts.NumUnits)*cfg.LabelsPerUnit, status.NumLabelsWritten)
 }
+
+// func TestBuilder_findCommitmentAtx_UsesLatestAtx(t *testing.T) {
```

Comment:
TODO: update tests after moving commitment ATX selection logic from builder to postsetupmanager